extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -734,7 +734,8 @@ public static void addCookieHeader(final Map<String, List<String>> headers) {
     }
 
     public static String generateConsentCookie() {
-        return CONSENT_COOKIE + 100 + numberGenerator.nextInt(900);
+        final int statusCode = 100 + numberGenerator.nextInt(900);
+        return CONSENT_COOKIE + statusCode;
```

Comment:
How is the new code different from the old?
<sub>Please don't kill me if it's obvious</sub>

Reply:
`CONSENT_COOKIE` is a String. That's why the result looks like `CONSENT=100580`.

Reply:
So if I understand correctly, an equivalent change would be keeping the method with one statement, but wrapping `100 + numberGenerator.nextInt(900)` in parenthesis, right?